src/scenes/Languages/Edit/EditLanguageForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { UpdateLanguageInput } from '../../../api';
+import {
+  DialogForm,
+  DialogFormProps,
+} from '../../../components/Dialog/DialogForm';
+import { SubmitError, TextField } from '../../../components/form';
+import { LanguageQuery } from '../Overview/LanguageOverview.generated';
+
+export type EditLanguageFormProps = DialogFormProps<UpdateLanguageInput> & {
+  languageFormValues: UpdateLanguageInput;
+  language?: LanguageQuery['language'];
+};
+
+export const EditLanguageForm = (props: EditLanguageFormProps) => {
+  const { language, languageFormValues } = props;
+
+  return (
+    <DialogForm
+      DialogProps={{
+        fullWidth: true,
+        maxWidth: 'xs',
+      }}
+      SubmitProps={{ disabled: !language }}
```

Comment:
Ah I see why you added `disabled` on `SubmitButton` for here. Can we just not allow the dialog to be opened while the language is loading?